source/_integrations/fibaro.markdown

```diff
@@ -29,29 +29,29 @@ ha_config_flow: true
 ha_integration_type: hub
 ---
 
-The [Fibaro](https://fibaro.com/) hub is a controller mainly connecting to Z-Wave devices.
+The Fibaro integration allows you to connect Home Assistant to a Fibaro Home Center or a Yubii Home hub so that you can control and monitor the connected devices and run scenes. Home Assistant uses a local connection to connect to the Fibaro hub.
 
-There is currently support for the following device types within Home Assistant:
+# Supported hub models
+Fibaro Home Center 2, Home Center Lite, Home Center 3, Home Center 3 Lite and Yubii Home.
 
+# Features
+1. Control devices connected to the hub and synchronize the state of the devices (see platforms for supported devices and capabilities).
+2. Entities are automatically added when configuring the Fibaro integration and upon restart of Home Assistant when changed in Fibaro hub.
+3. Support for multiple hubs.
+4. Activate scenes defined in the Fibaro hub.
+
+{% include integrations/config_flow.md %} 
+
+<div class='note'>
+It is recommended to assign a static IP address to your Fibaro controller. This ensures that it won't change its IP address, so you won't have to change the `url` if the controller reboots and comes up with a different IP address. See your router's manual for details on how to set this up. If you need the MAC address of your Fibaro, check the label on the bottom.
+</div>
```

Comment:
```suggestion
<div class='note'>

It is recommended to assign a static IP address to your Fibaro controller. This ensures that it won't change its IP address, so you won't have to change the `url` if the controller reboots and comes up with a different IP address. See your router's manual for details on how to set this up. If you need the MAC address of your Fibaro, check the label at the bottom.

</div>
```

Reply:
Why do we have this note even? Why don't we detect updates of the IP address and reconnect automatically?

We should not have to burden users with these details if not needed.

../Frenck

Reply:
The only thing Fibaro does is providing a mDNS entry. As I understand this is not enough to be discovered by HA.

Reply:
Home Assistant supports discovery on mDNS (zeroconf), including updates.
Additionally, we can listen to DHCP requests from devices and updated config entries based on those responses.